roles/apache/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
 - include: apache.yml
+  when: ansible_os_family == "Debian"
```

Comment:
nit: this seems reasonable, but if we're going to do apache_redhat can we rename this apache_ubuntu

Reply:
I'll rename to redhat.yaml to match the later patches (as you can tell this was an earlier patch. Also later I abandoned apache and used nginx, so I'm not 100% sure this even works.. I'm also not sure why we have apache still)